### DISCUSSION TITLE
Recommendations For Antibodies for Pig/Swine FFPE Tissues

### DISCUSSION THREAD

**Discussion by bschakib** [IBEXImagingCommunity/ibex_imaging_knowledge_base#228] [Q&A]
Hi guys,
I am just wondering if anyone has recommendations for the following antibodies: CD4, CD8, CD20, CD31 and CD68 that will work on FFPE Pig/Swine tissue sections?
Best regards
Chakib

**Comment by radtkea** [comment]
Hi Chakib,

Thank you very much for asking your question in the open! This is a great question. I will be sure to share with my colleagues who can assist you as well.

First, I use [CiteAb](https://www.citeab.com/) to find antibodies. 

Second, this [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4823807/) screens several antibodies in pig/swine FFPE tissues and might be helpful. 

Third, I would check different vendor sites. Thermo has an extensive catalog of antibodies directed against "exotic" species. See this [query](https://www.thermofisher.com/antibody/primary/query/CD8/filter/species/Pig) for an anti-CD8 antibody. 

I hope this helps! We would love to know what works and doesn't work for you.

Best wishes,
A

**Reply by radtkea** [reply]
@DasEFX130 Do you have any suggestions Shelley? Thanks so much!

**Reply by bschakib** [reply]
Hi Andrea,

Thank you for your prompt and detailed response! I really appreciate the resources you provided. I’ll take a look at CiteAb as well as the paper you referenced for pig/swine FFPE tissues. Thermo’s catalog sounds promising, and I’ll definitely explore the anti-CD8 antibody you mentioned.

Once I’ve had the chance to test a few options, I’ll share my feedback on what worked best.

Thanks again for your support!

Best,
Chakib

**Reply by bschakib** [reply]
Hi Andrea,
This paper omits the vender information of the antibodies. Is there any way how to figure out the ordering information of these antibodies?
Your help is greatly appreciated!
Best regards,
Chakib

**Reply by radtkea** [reply]
Hi Chakib,

Oh yes, that's really too bad and why we need [RRIDs](https://pmc.ncbi.nlm.nih.gov/articles/PMC5854161/)! For the monoclonal antibodies, you can search by clone ID. For example, the antibody against actin, aortic smooth muscle is reported as 1A4. If you use the dropdown menu for the reagent resources and select "1A4", then you will see that we have 26 entries using this clone. You can find purchase information (Vendor, catalog number) for each clone in the knowledge-base.

![Screenshot 2024-10-22 084216](https://github.com/user-attachments/assets/a0816f71-fb66-4987-8863-df08b13635b8)

You were specifically interested in: CD4, CD8, CD20, CD31 and CD68. Unfortunately, all of the monoclonal antibodies tested by the authors were reported to not work. 

I recommend you email the corresponding author:
![image](https://github.com/user-attachments/assets/a0c4b77c-a5c9-495a-9dab-d67446725a26)

I will follow up with a few more people in the antibody world.

Best wishes, 
Andrea

**Reply by tashkar** [reply]
Hello,
we are trying early next week a bunch of antibodies (around 40) on a paraffin pig kidney tissue section with CODEX/Phenocycler run on the same slide as a human kidney- I will post here what worked in pig tissue once we validate data- Stay tuned
Best wishes,
Tarek

**Comment by bschakib** [comment]
Dear Andrea,
Hope you are doing very well.... Thank you for the information, it's really helpful!
I appreciate your continued support!
Best Regards,
Chakib

**Reply by radtkea** [reply]
Hi Chakib,

You're very welcome. Happy that it is helpful.

I hope we can, collectively, find some answers for you:)

Best wishes,
Andrea

**Reply by bschakib** [reply]
Thank you so much Andrea! I really appreciate the support. I will keep you updated as I make progress.
Best regards
Chakib

**Comment by DasEFX130** [comment]
Hey Andrea and Chakib,
Sorry for the delay! Thanks Andrea for the detailed method through the community. That database is helpful with all the information. Chakib: One company that has a number of pig/swine clones is Bioss (https://www.biossusa.com/products/bs-0468r) and they have a trial size available too. They have data listed on their website for colormetric applications so you may need to test if it works for your purposes.  I took a look through the Thermo site too since there is a swine/pig catalogue of antibodies. Many of the immune cell markers for pig have been tested in flow cytometry applications but not yet for tissue staining/IHC. Hope that helps!

**Reply by bschakib** [reply]
Dear Shelley,
Thank you so much for the update, I will check out the Bioss and the trial size sounds a good option for initial testing. So, I will follow up once I have tested a few antibodies.
I really appreciate the suggestion and the support.
Best
Chakib

**Comment by DasEFX130** [comment]
Best of luck!

**Reply by radtkea** [reply]
Wonderful, thank you Tarek for sharing your knowledge with us! Fingers crossed!

**Reply by bschakib** [reply]
Sounds exciting! I look forward to seeing which ones work well with the pig tissue. Thanks for keeping us updated, and good luck with the validation!
Best wishes
Chakib
